core/state/iterator.go

```diff
@@ -62,6 +61,9 @@ func (it *NodeIterator) Next() bool {
 	// Otherwise step forward with the iterator and report any errors
 	if err := it.step(); err != nil {
 		it.Error = err
+		if it.Error != nil {
+			fmt.Printf("error: %v\n", it.Error)
```

Comment:
use `log.Error`?

Reply:
This is an print for local debugging, I have deleted it